### PR TITLE
Update REDIS environment variable

### PR DESCRIPTION
**Pull request by luisgmetzger**
## Summary
- This PR changes the REDIS_HOST environment variable to REDIS_SYSTEM_URL
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [ ] You have specified at least one "Reviewer".

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -147,9 +148,7 @@ gem 'mail', '~> 2.7.1'
 gem 'feedjira', '~> 3.2'
 gem 'bootsnap', require: false
 gem 'rails_semantic_logger', '~> 4.14'
-gem 'dotenv', '~> 3.1'
 gem 'whenever', '~> 1.0', require: false
-gem 'puma', '~> 5.6'
```
Comment (stevenbarragan) [inline Gemfile:152]: We need puma 5.6 on all environments. Can you revert your changes to the Gemfile? They're not related to the redis issue.

File: config/environments/production.rb [hunk 2]
```diff
@@ -84,12 +84,6 @@
   # Don't log any deprecations.
   config.active_support.report_deprecations = false
 
-  # Use a different logger for distributed setups.
-  # require "syslog/logger"
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new "app-name")
-
-  config.rails_semantic_logger.format = :json
```
Comment (stevenbarragan) [inline config/environments/production.rb:91]: We still need this. Please put it back.

File: config/initializers/load_resque.rb [hunk 3]
```diff
@@ -7,10 +7,9 @@
 require 'resque/server'
 require 'resque/job_timeout'
 
-host = ENV['REDIS_HOST']
-port = ENV['REDIS_PORT']
+redis_url = ENV['REDIS_SYSTEM_URL']
 
-Resque.redis = "#{host}:#{port}"
+Resque.redis = "#{redis_url}"
```
Comment (stevenbarragan) [inline config/initializers/load_resque.rb:12]: Do not interpolate
```ruby
Resque.redis = redis_url
```

Even bettery
```ruby
Resque.redis = ENV['REDIS_SYSTEM_URL']
```

File: app/models/rtu_dashboard.rb [hunk 4]
```diff
@@ -109,3 +109,6 @@ def extract_significant_terms(buckets)
   end
 
 end
+
+
+2 > 1
```
Comment (stevenbarragan) [inline app/models/rtu_dashboard.rb:114]: 👀

**Review by stevenbarragan — CHANGES_REQUESTED** [review]
We can't get this merged!

Your Gemfile changes break staging, and you removed our setting to log as json that we need in production. This kind of issues should not make it to review.

Can you please take a look at your changes before sending your pull request, make sure they work and that we are not losing (or adding) any unrelated features please?

File: Gemfile (hunk 1 above)
Comment (luisgmetzger) [reply Gemfile:152]: @stevenbarragan  - There seems to be duplicates between all branches. I found puma within development, test blocks and outside of those blocks in the Gemfile. I will put this one back but this Gemfile will still have a change to remove the dups.

**Comment by luisgmetzger** [comment]
@stevenbarragan  - I have addressed your comments and ready for another review.

**Review by stevenbarragan — APPROVED** [review]

**Pull request merged**